### ISSUE TITLE
logging is not giving spring beans

### ISSUE THREAD

**Issue by jitendrazkumar** [GoogleCloudPlatform/spring-cloud-gcp#1873]
I want to use the spring beans from logging module for my customized purpose but looks like logging module does not create spring beans .

**Comment by meltsufin** [comment]
That's correct. Our logging module provides support for Logback and Trace, which doesn't require creation of Logging client library beans like in other modules. Are you basically looking to use the Logging API directly?
